framework/core.py

```diff
@@ -216,6 +217,12 @@ def initialise_framework(self, options):
         self.start_proxy(options)  # Proxy mode is started in that function.
         # Set anonymised invoking command for error dump info.
         self.error_handler.SetCommand(OutputCleaner.anonymise_command(command))
+        # upload tool report
+        if options['UploadTool']:
+            pUploader = PluginUploader(options['UploadTool'])
+            pUploader.init_uploader(options['ReportUploadPath'])
```

Comment:
It has been already taken care [Line 168](https://github.com/DoomTaper/owtf/blob/uploader/owtf.py#L168)